### PR TITLE
release: bump to 0.17

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,25 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.16.0] - 2017-01-29
+## [0.16.0] - 2018-04-03
```
Comment (lebauce) [inline CHANGELOG.md]: 0.17.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,25 @@
...
+### Added
+- Add Latency to WebUI topology links
+  Packet Injector now allows to increment ICMP `id` for each packet
+  New `Light` WebUI theme
+  Add `Has` Gremlin step to `SocketInfo` step allowing to filter socket information
```
Comment (lebauce) [inline CHANGELOG.md]: - New `socketinfo` probe to retrieve active sockets of a host. The new `Sockets` Gremlin step can be used to retrieve socket information corresponding to flows.

**Comment by safchain** [comment]
rerun tests

**Comment by nplanel** [comment]
need #927

**Comment by safchain** [comment]
rerun selenium-tests

**Pull request merged**